### PR TITLE
Document ebsnvme-id on FreeBSD

### PR DESCRIPTION
**Pull request by cperciva**
*Issue #, if available:*

*Description of changes:*

Document ebsnvme-id on FreeBSD.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/nvme-ebs-volumes.md [hunk 1]
```diff
@@ -91,6 +91,19 @@ Volume ID: vol-01324f611e2463981
 
 Amazon Linux also creates a symbolic link from the device name in the block device mapping \(for example, `/dev/sdf`\), to the NVMe device name\.
 
+**FreeBSD AMIs**
```
Comment (colmmacc) [inline doc_source/nvme-ebs-volumes.md:94]: <3 FreeBSD and thank you for maintaining it! This paragraph looks good to me, but I do think it should come after the  "other linux AMIs" section, based purely on frequency of usage.

Comment (cperciva) [reply doc_source/nvme-ebs-volumes.md:94]: Yeah, I wondered about that.  In the end I figured that it made more sense to keep the platforms which use `ebsnvme-id` together; but I'm ok with moving this if Amazon would prefer to put FreeBSD at the end.

(Of course, if you want to go *completely* based on frequency of usage, I think "Other Linux" would come before "Amazon Linux"...)

Comment (cperciva) [reply doc_source/nvme-ebs-volumes.md:94]: #114

**Comment by julieso** [comment]
Thanks for your contribution! "Other Linux AMIs" is a bit of a catch-all, so it should be at the end. Therefore, I went with this pull request (after a few edits) instead of #114

**Pull request merged**